plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoConnectorTest.java

```diff
@@ -18,17 +18,17 @@
 import com.mongodb.MongoClient;
 import com.mongodb.client.MongoCollection;
 import io.trino.sql.planner.plan.LimitNode;
-import io.trino.testing.AbstractTestIntegrationSmokeTest;
-import io.trino.testing.MaterializedResult;
-import io.trino.testing.MaterializedRow;
-import io.trino.testing.QueryRunner;
+import io.trino.testing.*;
```

Comment:
We are disallowing `*` import. I would recommend setting code-style of 
https://github.com/trinodb/trino/blob/master/DEVELOPMENT.md#code-style

Reply:
I'm not doing that on purpose, seems IDEA done something automatically, sorry.
I will change it later